benefit/data/sheets/CaseEventToFields/CaseEventToFields-sandl-nonprod.json

```diff
@@ -192,7 +192,60 @@
     "PageFieldDisplayOrder": 1,
     "PageID": "Notice_Staging",
     "PageLabel": "Notice Document",
-    "PageShowCondition": "actionSetAside.action = \"refuse\"",
+    "PageShowCondition": "setAside.action = \"refuse\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "requestPostHearing",
+    "CaseFieldID": "requestPostHearingTypeSelected",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "23/11/2022",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "Type_Selection",
+    "PageLabel": "Application Type Selection",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "requestPostHearing",
+    "CaseFieldID": "setAside",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "COMPLEX",
+    "LiveFrom": "23/11/2022",
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 1,
```

Comment:
How do these display orders work? Is it ok to have some duplicates?

Reply:
It's best not to, but in this case, the field is on page 2: `"PageDisplayOrder": 2,` so there is no duplication